### PR TITLE
Rspec testing

### PR DESCRIPTION
**Pull request by AlejandroNeira98**
### Project requirements
- Add unit tests to your classes:
  - `Book`
  - `Person`
  - `Student`
  - `Teacher`
  - `Classroom`
  - `Rental`

### CODE REVIEW

File: spec/rental_spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      person_name = @rental.person.name
+      expect(person_name).to eql 'Neira'
+    end
+
+    it 'Should have book named Abc' do
+      book_title = @rental.book.title
+      expect(book_title).to eql 'Abc'
+    end
+  end
+end
```
Comment (Whoistolu) [inline spec/rental_spec.rb]: - [ ] [REQUIRED] Hello Team, great job creating unit tests for all the methods in all the classes, however, the `rental_spec.rb` test fails. The test looks good, but I do not really know why it fails.  Let's ensure we fix this to get this project approved. 
Screenshot from 2022-05-25 22-06-22

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi Team,
@AlejandroNeira98 and @Saadat123456 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/rental_spec.rb (hunk 1 above)
Comment (saadat-ali-awan) [reply spec/rental_spec.rb]: Can you kindly run it again? It is working fine on the device of all the partners.

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @AlejandroNeira98 and @Saadat123456     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**